.eslintignore

```diff
@@ -2,5 +2,6 @@ build
 build-module
 coverage
 node_modules
+packages/block-serialization-spec-parser
```

Comment:
Why do we ignore the entire package? Could we just ignore the single files which are generated? The test file, for example, includes a number of legitimate code style violations.

Reply:
Haha, I know what happened. I used `toMatchInlineSnapshot` which uses prettier behind the scenes. We can't really use it with the current setup unless we adopt `calypso-prettier` :)